stories/axis-boundaries.stories.tsx

```diff
@@ -12,10 +12,10 @@ export default {
     xMax: 6,
     yMin: 0,
     yMax: 6,
-    paddingLeft: 0.01,
-    paddingRight: 0.01,
-    paddingTop: 0.01,
-    paddingBottom: 0.01,
+    paddingLeft: '1%',
```

Comment:
I could not let work this example 

http://localhost:6006/?path=/story/plot-axis-boundaries--control&args=paddingLeft:1

and in contrast with other examples like

http://localhost:6006/?path=/story/api-axis--axis-bottom

 we don't see in the storybook that the property may be either a number of a string.

Reply:
Yeah, I will add that to the breaking changes too and remove the controls from this story. It's not possible to specify both paddings and min/max, as we discussed because they conflict with each other, especially when zoom interactions enter the game.